engine/apps/alerts/models/resolution_note.py

```diff
@@ -83,6 +83,11 @@ class ResolutionNoteSlackMessage(models.Model):
     class Meta:
         unique_together = ("thread_ts", "ts")
 
+        indexes = [
+            models.Index(fields=["ts", "thread_ts", "alert_group_id"]),
+            models.Index(fields=["ts", "thread_ts", "slack_channel_id"]),
+        ]
+
```

Comment:
added indices to speed up querying these objects. These are the two filter patterns we use when querying these objects.